PWGEM/PhotonMeson/DataModel/gammaTables.h

```diff
@@ -363,6 +363,8 @@ DECLARE_SOA_INDEX_COLUMN_FULL(MatchedTrack, matchedTrack, int, Tracks, "_Matched
 DECLARE_SOA_COLUMN(X, x, float);                                                    //! cluster hit position in ALICE global coordinate
 DECLARE_SOA_COLUMN(Y, y, float);                                                    //! cluster hit position in ALICE global coordinate
 DECLARE_SOA_COLUMN(Z, z, float);                                                    //! cluster hit position in ALICE global coordinate
+DECLARE_SOA_COLUMN(CellX, cellx, int);                                              //! cell index x of cluster hit position
+DECLARE_SOA_COLUMN(CellZ, cellz, int);                                              //! cell index z of cluster hit position
```

Comment:
Are these two columns really needed?

Reply:
yes. this is needed to see hit map on PHOS.